rapport/index.Rmd

```diff
@@ -0,0 +1,169 @@
+Daarom moet een wildbeheereenheid (WBE) die op patrijs wenst te jagen, eerst inventariseren waar er allemaal patrijzen rondlopen.
+Op basis van deze waarnemingen bepaalt het Instituut voor Natuur- en Bosonderzoek (INBO) het aantal broedparen in elke WBE.
+Tenslotte delen we dit aantal broedparen door de oppervlakte open ruimte om tot de dichtheid van patrijs (broedparen per 100 ha open ruimte) te komen.
+Om deze oppervlakte te kennen, hebben we nood aan een kaart die bepaalt wat open ruimte is.
+
+**Waarom een nieuwe open ruimte kaart?**
+
+Ne de tellingen van voorjaar 2021 lieten meerdere jagers ons weten dat de gebruikte open ruimte kaart niet altijd overeenkomt met de werkelijkheid.
+Sommige percelen zijn open ruimte maar bebouwd of bebost.
+Waardoor de oppervlakte open ruimte overschat en de dichtheid van patrijs onderschat is.
+
+De vorige versie van de open ruimte kaart is gebaseerd op de meest recente publicatie van de biologische waarderingskaart (BWK).
+Hoewel deze versie in 2020 gepubliceerd is, wil dat niet zeggen dat alle informatie uit 2020 stamt.
+In de praktijk bleek zowat driekwart ouder dan 2007.
+Vandaar dat we op zoek gaan naar een betere bron.
+
+**Een _open_ open ruimte kaart**
+
+Na het afwegen van een aantal mogelijkheden, kozen we [OpenStreetMap](https://www.openstreetmap.org) als bronkaart.
+Net zoals alle andere kandidaten, is deze kaart niet perfect.
+Ze is evenwel goed genoeg om te gebruiken.
+En heeft als troef dat fouten relatief eenvoudig en snel te verbeteren zijn.
+Bij de start van de inventarisatie krijgen de jagers een voorlopige versie van de open ruimte kaart.
+Wanneer een jager een fout in de open ruimte kaart vaststelt, kan deze de fout melden of zelf aanpassen in OpenStreetMap.
+Na het berekenen van het aantal broedparen kunnen we de finale versie van de open ruimte kaart op basis van de meest recente informatie uit OpenStreetMap.
+
+Dit is een belangrijk verschil met de overige bruikbare kaarten.
+Deze beschikken niet over een mogelijkheid om ze bij te werken.
+Verder zijn ze gebaseerd om informatie die minstens 1 tot 2 jaar oud is.
+
+Voor de jagers biedt deze manier van werken hun de mogelijkheid om zelf bij te dragen aan een correcte open ruimte kaart.
+Aangezien OpenStreetMap de informatie met open gebruiksvoorwaarden aanbiedt, kan iedereen genieten van een verbeterde OpenStreetMap kaart.
+
+**Een [reproduceerbare](https://nl.wikipedia.org/wiki/Reproduceerbaarheid) open ruimte kaart**
+
+In dit rapport beschrijven we hoe we de informatie van OpenStreetMap omzetten naar een open ruimte kaart.
+De omzetting hebben we als een computerprogramma uitgewerkt.
+We publiceren die computerprogramma samen met de open ruimte kaart en de gebruikte bronkaarten (OpenStreetMap en grenzen jachtgebieden).
+Op die manier kan iedereen de gebruikte regels (laten) nakijken of zelf uitvoeren.
+
+\benglish
+
+# English abstract {-}
+
+Insert a translation of the summary here.
```

Comment:
**Why do we need an open space map?**

The Agency for Nature and Forests (ANB) only allows the hunt for partridges in an area if several conditions are met.
One of these conditions is that there are enough partridges present.
For this, ANB applies a minimum of 3 breeding pairs per 100 ha of open space.
Therefore, a wildlife management unit (WMU) that wishes to hunt partridge must first make an inventory of where all the partridges are.
Based on these observations, the Research Institute for Nature and Forest (INBO) determines the number of breeding pairs in each WMU.
Finally we divide this number of breeding pairs by the area of open space to arrive at the density of partridge (number of breeding pairs per 100 ha open space).
To obtain the area of open space in each WMU, we need a map that defines what open space is.

**Why a new open space map?

After the spring 2021 censuses, several hunters informed us that the open space map that was used does not always correspond to reality.
Some plots were classified as open space on the map but were but cultivated or forested in reality.
This overestimated the area of open space and underestimated the density of partridge.
The previous version of the open space map is based on the most recent publication of the biological valuation map (BWK).
Although this version of BWK was published in 2020, that does not mean that all the information was from 2020.
actually, about three quarters of the information turned out to be older than 2007.
Hence, we are looking for a better source.

**An _open_ open space map**

After weighing a number of options, we chose [OpenStreetMap](https://www.openstreetmap.org) as our source map.
Like all the other candidates, this map is not perfect.
However, it is good enough to use.
And it has the advantage that errors are relatively easy and quick to correct.
At the start of the surveys, hunters are given a preliminary version of the open space map.
When a hunter identifies an error in the open space map, they can report the error or modify it themselves in OpenStreetMap.
After calculating the number of breeding pairs, we can provide the final version of the open space map based on the most recent information from OpenStreetMap.
This is an important difference between OpenStreetMap and other usable maps.
The others do not have an option to update them.
Furthermore, they are based on information that is at least 1 to 2 years old.

For hunters, this way of working gives them the opportunity to contribute to a correct open space map themselves.
Since OpenStreetMap provides the information with open use terms, everyone can enjoy an improved OpenStreetMap.

**A [reproducible](https://nl.wikipedia.org/wiki/Reproduceerbaarheid) open space map**

In this report, we describe how to convert the information from OpenStreetMap to an open space map.
We established the conversion as a computer program.
We publish that computer program together with the open space map and the source maps that were used (OpenStreetMap and hunting area boundaries).
In this way, everyone can check the rules (or have them checked), use, or implement them themselves.